### DISCUSSION TITLE
Single Channel Dimmer

### DISCUSSION THREAD

**Discussion by kivanc87** [RavenSystem/esp-homekit-devices#1995] [MEPLHAA Scripts]
Hello, I would be very happy if you could help me make the single-channel dimmable button code that I need.

**Comment by kivanc87** [comment]
{"c":{"b":[{"g":0,"t":5}]},"a":[{"t":30,"g":[1],"b":[{"g":0}]},{"t":30,"g":[2],"b":[{"g":0}]},{"t":30,"g":[3],"b":[{"g":0}]}]}


my old commands were like this..

**Reply by RavenSystem** [reply]
{"c":{"io":[[[0],6],[[1,2,3],7]],"b":[{"g":0,"t":5}]},"a":[{"t":30,"g":[1],"b":[{"g":0}]},{"t":30,"g":[2],"b":[{"g":0}]},{"t":30,"g":[3],"b":[{"g":0}]}]}

**Reply by kivanc87** [reply]
thank you for your attention.


{"c":{"io":[[[2],7]]},"b":[{"g":0,"t":5}],"a":[{"t":30,"g":[2]}]}

I couldn't get the button to work on this. Can you help with this?

**Reply by RavenSystem** [reply]
{"c":{"io":[[[0],6],[[2],7]]},"b":[{"g":0,"t":5}],"a":[{"t":30,"g":[2]}]}

**Reply by kivanc87** [reply]
no, the button is useless

**Reply by RavenSystem** [reply]
{"c":{"io":[[[0],6],[[2],7]]},"b":[{"g":0,"t":5}],"a":[{"t":30,"g":[2],"b":[{"g":0}]}]}

**Reply by kivanc87** [reply]
Ok. it's done now thanks a lot
